apps/04TowersOfHanoi.js

```diff
@@ -18,31 +18,44 @@ function printStacks() {
   console.log("b: " + stacks.b);
   console.log("c: " + stacks.c);
 }
-
-function movePiece() {
+//Popping and pushing the startstack and endstack so the pieces will move.
+function movePiece(startStack, endStack) {
   // Your code here
+  var begin = stacks[startStack].pop();
```

Comment:
I like breaking it up here! I bet you can think of an even better variable name.